src/vos/vos_ts.h

```diff
@@ -62,6 +62,13 @@ struct vos_ts_pair {
 	struct dtx_id	tp_tx_rh;
 };
 
+struct vos_wts_cache {
+	/** write timestamps */
+	daos_epoch_t	wc_ts_w[2];
+	/** Index of max timestamp */
+	uint32_t	wc_w_max;
```

Comment:
Style: "/* Index of max timestamp in wc_ts_w */".

Reply:
done

Reply:
Curious what the style guideline for comments actually is. I see both '/**' and '/*' in the source.

Reply:
I don't know the "official" one, but my understanding is that documentation for APIs shall be Doxygen, whereas non-API comments (e.g., one in the middle of a function and one for a private field) shall not be Doxygen.

Reply:
Oh, my style comment above isn't about Doxygen, but "in wc_ts_w", as I initially didn't know it's an index of what data structure.